config/localTemplate.go

```diff
@@ -520,6 +520,17 @@ type Local struct {
 	// BlockServiceMemCap is the memory capacity in bytes which is allowed for the block service to use for HTTP block requests.
 	// When it exceeds this capacity, it redirects the block requests to a different node
 	BlockServiceMemCap uint64 `version[28]:"500000000"`
+
+	// P2PEnable turns on the peer to peer network
+	P2PEnable bool `version[28]:"false"`
```

Comment:
should the version be updated to 29 or later and update the Version field, too?

Reply:
I think we don't necessarily need to bump the version unless we're changing a default for a config value which was already in the config.

Reply:
Only reason I can think that we should add it, is because of the example config packaged into the earlier releases. There would be two different example files with the same version but different contents.

Reply:
Yeah I've been updating the version when introducing a new config value as well